[GB.QT5.WEBKIT]
* BUG: The component now should compile correctly with C++11.
Authored by:  gambas 2017-01-09
Browse code at this revision
Parent: [r8049]
Child: [r8051]
changed	/gambas/trunk/gb.qt5/src/opengl/Makefile.am
changed	/gambas/trunk/gb.qt5/src/webkit/Makefile.am
/gambas/trunk/gb.qt5/src/opengl/Makefile.am Diff Switch to side-by-side view
--- a/gambas/trunk/gb.qt5/src/opengl/Makefile.am
+++ b/gambas/trunk/gb.qt5/src/opengl/Makefile.am
@@ -6,6 +6,7 @@
 
 gb_qt5_opengl_la_LIBADD = @QT5OPENGL_LIB@ @QT5OPENGL_LIB@
 gb_qt5_opengl_la_LDFLAGS = -module @LD_FLAGS@ @QT5OPENGL_LDFLAGS@
+gb_qt5_opengl_la_CXXFLAGS = @THREAD_INC@ $(AM_CXXFLAGS) -std=c++11
 gb_qt5_opengl_la_CPPFLAGS = @QT5OPENGL_INC@
 
 gb_qt5_opengl_la_SOURCES = main.h main.cpp
/gambas/trunk/gb.qt5/src/webkit/Makefile.am Diff Switch to side-by-side view
--- a/gambas/trunk/gb.qt5/src/webkit/Makefile.am
+++ b/gambas/trunk/gb.qt5/src/webkit/Makefile.am
@@ -6,6 +6,7 @@
 
 gb_qt5_webkit_la_LIBADD = @QT5WEBKIT_LIB@
 gb_qt5_webkit_la_LDFLAGS = -module @LD_FLAGS@ @QT5WEBKIT_LDFLAGS@
+gb_qt5_webkit_la_CXXFLAGS = @THREAD_INC@ $(AM_CXXFLAGS) -std=c++11
 gb_qt5_webkit_la_CPPFLAGS = @QT5WEBKIT_INC@ -I$(top_srcdir)/share -I$(top_srcdir)/src/share
 
 gb_qt5_webkit_la_SOURCES = \
